pyvelox/pyvelox.h

```diff
@@ -29,6 +34,43 @@ namespace facebook::velox::py {
 
 namespace py = pybind11;
 
+struct PyVeloxContext {
```

Comment:
I think we want only a single instance, Can we delete for this class :
1 . default/ copy constructors
2. Move and copy assignment operators ?

Reply:
We can't delete the default constructor but we can make it private. Everything else we can delete

Reply:
Actually we can't even make default constructor private because `unique_ptr` won't see it.